src/components/new-safe/create/steps/StatusStep/useIsZkEvmChain.ts

```diff
@@ -0,0 +1,13 @@
+import useChainId from '@/hooks/useChainId'
+
+enum ZkChains {
+  ZKSYNC_ERA_MAINNET = '324',
+  POLYGON_ZKEVM = '1101',
+}
+
+// TODO: Remove when we update SDK
+export const useIsZkEvmChain = (): boolean => {
+  const chainId = useChainId()
+
+  return Object.values(ZkChains).includes(chainId as ZkChains)
+}
```

Comment:
I would suggest using a config-service feature toggle for this instead.

Reply:
I've updated it accordingly, as well as adding it to the staging config service. I also added a note to the rollout plans of these networks.